lib/docs/index.html

```diff
@@ -276,7 +271,7 @@
       .column {
         flex-basis: 0;
         flex-grow: 1;
-        min-width: 24rem;
+        min-width: min(24rem, 90%);
```

Comment:
This new change prevents the column from forcing a 24rem minimum width on small screens, where it could cause unnecessary x overflow. 90% seems to be a good value from my testing to avoid overflow while maintaining the two-column layout on larger screens.